airflow/ui/src/components/DataTable/DataTable.test.tsx

```diff
@@ -20,14 +20,14 @@
 import { describe, expect, it, vi } from "vitest";
 import { render, screen } from "@testing-library/react";
 import { DataTable } from "./DataTable.tsx";
-import { ColumnDef, PaginationState } from "@tanstack/react-table";
+import type { ColumnDef, PaginationState } from "@tanstack/react-table";
 import "@testing-library/jest-dom";
 
-const columns: ColumnDef<{ name: string }>[] = [
+const columns: Array<ColumnDef<{ name: string }>> = [
```

Comment:
Open to change this, it was changed from `Type[]` to `Array<Type>`. There are some scenarios in which the `Type[]` syntax can get awkward (having to use parenthesis), but I'm more than ok reverting this back.